catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/impl/OpenSearchParserImpl.java

```diff
@@ -341,34 +341,42 @@ private boolean hasParameter(String parameter, List<String> parameters) {
    *
    * @param lon latitude in decimal degrees (WGS-84)
    * @param lat longitude in decimal degrees (WGS-84)
-   * @param radius radius, in meters
+   * @param radius search radius, in meters
    * @return Array of bounding box coordinates in the following order: West South East North. Also
    *     described as minX, minY, maxX, maxY (where longitude is the X-axis, and latitude is the
    *     Y-axis).
    */
-  private double[] createBBoxFromPointRadius(double lon, double lat, double radius) {
+  private static double[] createBBoxFromPointRadius(
+      final double lon, final double lat, final double radius) {
     double minX;
     double minY;
     double maxX;
     double maxY;
 
-    double lonDifference = radius / (LAT_DEGREE_M * Math.cos(lat));
-    double latDifference = radius / LAT_DEGREE_M;
-    minX = lon - lonDifference;
-    if (minX < MIN_LON) {
-      minX += MAX_ROTATION;
-    }
-    maxX = lon + lonDifference;
-    if (maxX > MAX_LON) {
-      maxX -= MAX_ROTATION;
-    }
+    final double latDifference = Math.toDegrees(radius / RADIUS_OF_THE_EARTH_IN_METERS);
+    final double lonDifference =
+        Math.toDegrees(
+            Math.asin(Math.sin(Math.toRadians(latDifference)) / Math.cos(Math.toRadians(lat))));
```

Comment:
Here is outline of solving it out
![img_20180208_095428](https://user-images.githubusercontent.com/9013416/35987135-2ade0e26-0cb8-11e8-86a0-76c36a085e14.jpg)
![img_20180208_095438](https://user-images.githubusercontent.com/9013416/35987143-2ec74db8-0cb8-11e8-8f13-003999337e70.jpg)

Reply:
I'm not quite sure where we differed, but I think it might be that I included that d=2*search radius in radians. Here's my work leading up to `final double lonDifference = Math.toDegrees(Math.asin(Math.sin(Math.toRadians(latDifference)) /  Math.cos(Math.toRadians(lat))));`:
![image](https://user-images.githubusercontent.com/8041246/36002799-b5ad4d48-0ce8-11e8-98a7-6040a18f407b.png)

Reply:
![image](https://user-images.githubusercontent.com/8041246/36002888-0c8bba6e-0ce9-11e8-896a-7306739b784b.png)

Reply:
![image](https://user-images.githubusercontent.com/8041246/36002910-234c1884-0ce9-11e8-8550-ddb01cd684d8.png)
![image](https://user-images.githubusercontent.com/8041246/36002922-295543ae-0ce9-11e8-9a0c-d7014fb8972f.png)
![image](https://user-images.githubusercontent.com/8041246/36002930-2d999f8c-0ce9-11e8-84a5-3cbf725e1cdb.png)
![image](https://user-images.githubusercontent.com/8041246/36002939-33a7ec12-0ce9-11e8-8546-44f25220cea6.png)